client/src/components/BizCard/BizCard.jsx

```diff
@@ -37,7 +37,7 @@ export default function BizCard() {
   async function stripeCheckoutHandler() {
     const stripe = await stripePromise;
     const response = await fetch(
-      "http://localhost:4000/api/stripecheckout/create-checkout-session",
+      "http://localhost:4000/api/stripecheckout/checkoutsession",
```

Comment:
This code cannnot connect to server in AWS environment...

it will be work maybe.
```
const url = process.env.AWS_BACKEND_URL || localhost
`http://${url}:4000/api/stripecheckout/checkoutsession`
```